packages/ndla-ui/src/ContentCard/ContentCard.jsx

```diff
@@ -9,9 +9,21 @@ import BackgroundImage from '../BackgroundImage';
 
 const classes = BEMHelper('c-content-card');
 
-const ContentCard = ({ heading, images, description, type, url, isFilm }) => (
+const ContentCard = ({
+  heading,
+  images,
+  description,
+  type,
+  url,
+  isFilm,
+  urlTarget,
+}) => (
   <article {...classes()}>
-    <SafeLink to={url} {...classes('link')}>
+    <SafeLink
+      to={url}
```

Comment:
Maybe spread a toLinkProps function instead (i.e. `<SafeLink {...toLinkProps()} />` .

Reply:
Is that a good idea? I feel that you could easily loose control over what props that are sent down. 

However, I see the benefit with a reusable function.